### ISSUE TITLE
 Parallelism

### ISSUE THREAD

**Issue by impulsecorp** [jhfjhfj1/autokeras#18]
I have your /examples/mnist.py file working on an AWS GPU server. I needed to delete the "time_limit=12 * 60 * 60" part though because that was giving an error. My question is, does it use multiple cores by default? I read the Parallelism section of your paper, but am unclear if I have to do anything to make it parallel. What if for example I want to run it on a CPU-only server with 128 cores, would it automatically use all of them? Also, when you say it runs multi-GPU, do you mean within the same machine, or can it also run over a cluster of multiple GPU servers?

Also, in Table 1 of your paper, where you compare Auto-Keras with similar programs, are those accuracy results on the test set or validation set? In your github repo it seems to be only split into training and test sets. Should that also include a validation set?

**Comment by impulsecorp** [comment]
Ignore my question about the accuracy results. I see now it is scored on the test set. So my only question is about parallelism.

**Comment by impulsecorp** [comment]
I tried it on a CPU server now with 40 cores, and it seemed to be using all of them automatically.
